fvm/state/accounts_test.go

```diff
@@ -9,14 +9,25 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-func TestAccounts_GetWithNoKeys(t *testing.T) {
-	chain := flow.Mainnet.Chain()
+func TestAccounts_Create_Sets_Registers(t *testing.T) {
+	ledger := state.NewMapLedger()
+
+	accounts := state.NewAccounts(ledger)
+	address := flow.HexToAddress("01")
+
+	err := accounts.Create(nil, address)
+	require.NoError(t, err)
 
+	require.Equal(t, len(ledger.RegisterTouches), 3) // exists + code + key count
+}
+
```

Comment:
It'd be good to add a test case for an already-existing account and confirm that it fails